deploy/eck-beats/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: eck-beats
+description: A Helm chart to deploy Elastic Beats managed by the ECK Operator.
+kubeVersion: ">= 1.20.0-0"
```

Comment:
I updated this to include the reasoning behind this requirement, as I think it's important.  This would need to be updated during the normal release process for the eck-operator, when the requirement changes.

Reply:
If a user is using a version of the operator that still supports a previous k8s version (for example ECK `2.2`  which still supports K8S `1.19`), but the user needs to update the Helm Chart: does it prevent the user to use that new version of the Chart on `1.19`? I'm just trying to understand how the min. k8s version supported by the operator and the min. k8s version supported by a Chart are tied.

Reply:
If a user was running eck `2.2` on K8S `1.19` and attempted to use these Helm Charts, they would fail (helm would throw a failure) because of this K8S version requirement in the Chart.yaml.